### PR TITLE
Add notes about local STAC files

### PR DESCRIPTION
**Pull request by drwelby**
Addresses #70 and #100 with some information on how to use STAC Browser with local files.

### CODE REVIEW

**Comment by m-mohr** [comment]
Thank you. Just want to quickly let you know that I've seen this, but don't have the time yet to review in detail.

Two quick thoughts:
1. I think this should be a separate page where we can also add more details, e.g. a JS variant would be good as you likely have the JS tooling for STAC Browser installed anyway (e.g. using the http-server package)
2. We need to add a note that in some cases (abs. links) you need to set the proxy url, otherwise links will be broken.
3. This is going towards v2, we should also migrate this to v3.

**Comment by drwelby** [comment]
I'll move this out to its own file, or if there's interest I could set up more formal docs with MkDocs.

**Comment by m-mohr** [comment]
Thanks. Right now the STAC Browser documentation is pretty lightweight and I think Markdown on GH is good enough, but once we have a more stable state of STAC Browser v3 + more docs this might be a good addition.

File: README.md [hunk 1]
```diff
@@ -33,8 +33,7 @@ To browse your own, set the `CATALOG_URL` CLI parameter when running the dev ser
 ```bash
 npm start -- --open --CATALOG_URL="http://path/to/catalog.json"
 ```
-
-To open a local file on your system, see [Using Local Files]( #using-local-files) below.
+To open a local file on your system, see `local_files.md`.
```
Comment (m-mohr) [inline README.md:36]:
```suggestion
To open a local file on your system, see the chapter [Using Local Files](local_files.md).
```

Let's make it a link.

File: local_files.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Using Local Files
+
+Web browser security settings prevent web pages from accessing local files using the normal request model. The solution is to serve the files from a local webserver. The following examples show `catalog.json` as the name of the local catalog you wish to browse - substitute the name of your catalog file if it is different.
+
+## Using Stac Browser from source
```
Comment (m-mohr) [inline local_files.md:5]:
```suggestion
## Using STAC Browser from source
```

File: local_files.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+```js
+http-server -p 8000 --cors
+```
+
+You can then use the option `--CATALOG_URL="http://localhost:8000/catalog.json"` when starting STAC Browser.
+
+If your catalog uses absolute file locations, you can use the `STAC_PROXY_URL` option to convert local file locations to local web browser URLs. For instance, if you are serving a catalog on the local file system at /home/user/catalog.json, but want to serve the data out from a server located at http://localhost:8000/, you can use `--STAC_PROXY_URL="/home/user|http://localhost:8000"`.
+
+## Using Stac Browser somewhere else
```
Comment (m-mohr) [inline local_files.md:25]:
```suggestion
## Using STAC Browser somewhere else
```

File: local_files.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+If you are using a hosted version of Stac Browser, such as the [Radiant Demo](https://radiantearth.github.io/stac-browser/#/), you can run a local webserver *if your STAC files use relative URIs*. One possibility is to use the Node package described above. Another is to use this Python 3 script:
```
Comment (m-mohr) [inline local_files.md:27]:
```suggestion
If you are using a hosted version of STAC Browser, such as the [Radiant Demo](https://radiantearth.github.io/stac-browser/#/), you can run a local webserver *if your STAC files use relative URIs*. One possibility is to use the Node package described above. Another is to use this Python 3 script:
```

**Review by m-mohr — APPROVED** [review]
Some minor comments, but otherwise lgtm. Thank you!

**Pull request merged**